### ISSUE TITLE
rebar3 compile error in master branch, tag: 2.2.0, 2.1.1

### ISSUE THREAD

**Issue by yaocl** [erlyaws/yaws#495]
compilation using rebar3 has error below
rebar3 compilation has error in master branch, tag: 2.2.0, 2.1.1

tag: 2.1.0, 2.0.9 "rebar3 compile" are both ok

```
===> Compiling /root/git/yaws-master/c_src/epam.c
===> Uncaught error in rebar_core. Run with DIAGNOSTIC=1 to see stacktrace or consult rebar3.crashdump
===> Uncaught error: badarg
===> Stack trace to the error location:
[{re,replace,
     [[99,95,115,114,99,47,101,112,97,109,46,99,58,50,58,49,48,58,32,22196,
       37325,37679,35492,65306,112,97,109,95,97,112,112,108,46,104,65306,78,
       111,32,115,117,99,104,32,102,105,108,101,32,111,114,32,100,105,114,101,
       99,116,111,114,121,10,32,35,105,110,99,108,117,100,101,32,60,112,97,
       109,95,97,112,112,108,46,104,62,10,32,32,32,32,32,32,32,32,32,32,94,
       126,126,126,126,126,126,126,126,126,126,126,10,32232,35695,25554,26039,
       12290,10],
      "c_src/epam.c","/root/git/yaws-master/c_src/epam.c",
      [{return,list},global]],
     [{file,"re.erl"},{line,969},{error_info,#{module => erl_stdlib_errors}}]},
 {pc_compilation,exec_compiler,4,
                 [{file,"/root/git/yaws-master/_build/default/plugins/pc/src/pc_compilation.erl"},
                  {line,192}]},
 {pc_compilation,compile_each,5,
                 [{file,"/root/git/yaws-master/_build/default/plugins/pc/src/pc_compilation.erl"},
                  {line,135}]},
 {pc_compilation,'-compile_sources/2-lists^foldl/2-0-',3,
                 [{file,"/root/git/yaws-master/_build/default/plugins/pc/src/pc_compilation.erl"},
                  {line,96}]},
 {pc_compilation,compile_sources,2,
                 [{file,"/root/git/yaws-master/_build/default/plugins/pc/src/pc_compilation.erl"},
                  {line,96}]},
 {pc_compilation,compile_and_link,2,
                 [{file,"/root/git/yaws-master/_build/default/plugins/pc/src/pc_compilation.erl"},
                  {line,42}]},
 {pc_prv_compile,'-do/1-lc$^0/1-0-',2,
                 [{file,"/root/git/yaws-master/_build/default/plugins/pc/src/pc_prv_compile.erl"},
                  {line,53}]},
 {pc_prv_compile,do,1,
                 [{file,"/root/git/yaws-master/_build/default/plugins/pc/src/pc_prv_compile.erl"},
                  {line,54}]}]
```

rebar3 version:

```
rebar3 -v
rebar 3.23.0 on Erlang/OTP 25 Erts 13.2.2.7
```

**Comment by vinoski** [comment]
Try changing the tag for `port_compiler` in `rebar.config` to v1.15.0:

```
diff --git rebar.config rebar.config
index a4a213a2..9d544232 100644
--- rebar.config
+++ rebar.config
@@ -22,7 +22,7 @@
              {clean, "./scripts/rebar-pre-script clean"}]}.

 {plugins, [
-   {pc, ".*", {git, "https://github.com/blt/port_compiler.git", {tag, "v1.11.0"}}}
+   {pc, ".*", {git, "https://github.com/blt/port_compiler.git", {tag, "v1.15.0"}}}
 ]}.

 {provider_hooks, [
```

**Comment by yaocl** [comment]
master branch is fixed with port_compiler v1.15.0, thx.
